pkg/istioUtil/util.go

```diff
@@ -0,0 +1,26 @@
+package istioUtil
+
+import (
+	"github.com/hashicorp/go-version"
+	"github.com/stackrox/istio-cves/types"
+)
+
+// IstioIsAffected gets the fixed-by version for vStr in Istion vuln.
+func IstioIsAffected(vStr string, vuln types.Vuln) (bool, string, error) {
```

Comment:
can probably just rename to `IsAffected` since it's clear this is about Istio from the package name

Reply:
can we pass in a `version.Version` instead of a `string`?